main.py

```diff
@@ -278,22 +299,34 @@ def private(update, context, grp=False, the_id=None, isgrp="(PRIVATE)"):
     with open("files/interactions.txt", "a") as f1:
         inp = f"UTC+0 {update.message.date} {isgrp} {reply} {update.message.from_user.full_name}" \
               f" ({update.message.from_user.username}) SAID: {update.message.text}\n"
-        out = shanitext.capitalize()
```

Comment:
why

Reply:
Its already capitalized when you join it. See line 297